charts/newrelic-infrastructure-v3/README.md

```diff
@@ -50,8 +48,22 @@ Kubernetes: `>=1.16.0-0`
 | ksm | object | See `values.yaml` | Configuration for the Deployment that collects state metrics from KSM (kube-state-metrics). |
 | ksm.enabled | bool | `true` | Enable cluster state monitoring. Advanced users only. Setting this to `false` is not supported and will break the New Relic experience. |
 | ksm.resources | object | 100m/150M -/850M | Resources for the KSM scraper pod. Keep in mind that sharding is not supported at the moment, so memory usage for this component ramps up quickly on large clusters. |
-| kubelet | object | See `values.yaml` | Configuration for the DaemonSet that collects metrics from the Kubelet |
+| kubelet.affinity.nodeAffinity | object | `{}` |  |
+| kubelet.annotations | object | `{}` |  |
+| kubelet.config | object | `{}` |  |
 | kubelet.enabled | bool | `true` | Enable kubelet monitoring. Advanced users only. Setting this to `false` is not supported and will break the New Relic experience. |
+| kubelet.extraEnv | list | `[]` |  |
+| kubelet.extraEnvFrom | list | `[]` |  |
+| kubelet.extraVolumeMounts | list | `[]` |  |
+| kubelet.extraVolumes | list | `[]` |  |
+| kubelet.nodeSelector | object | `{}` |  |
+| kubelet.resources.limits.memory | string | `"300M"` |  |
+| kubelet.resources.requests.cpu | string | `"100m"` |  |
+| kubelet.resources.requests.memory | string | `"150M"` |  |
+| kubelet.tolerations[0].effect | string | `"NoSchedule"` |  |
+| kubelet.tolerations[0].operator | string | `"Exists"` |  |
+| kubelet.tolerations[1].effect | string | `"NoExecute"` |  |
+| kubelet.tolerations[1].operator | string | `"Exists"` |  |
```

Comment:
Someone forgot to update the README here :(

Reply:
Don't be passive-agressive when you know it was me... Fixing that was out of scope of that PR.

There is it still has the issue that commented variables are not templated so you have eliminated `lowDataMode` and `common.config.interval`. That was the reason I did the README manually.

Reply:
Let me see if I can whisper to `helm-docs` so it works nicely.

Reply:
Restored, please take a look @kang-makes :)